config/jobs-chromeos.yaml

```diff
@@ -357,6 +357,16 @@ jobs:
     template: baseline.jinja2
     kind: test
 
+  baseline-nfs-arm64-mediatek: &baseline-nfs-job
+    template: baseline-nfs.jinja2
+    kind: test
+    params: &baseline-nfs-job-params
+      nfsroot: http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/{arch}
```

Comment:
This will likely fail for x86, as `arch` is `x86_64` in the platform configs, whereas the corresponding subfolder here is `amd64`...

This is a mess, I know, see for another example of where it messes up: https://github.com/kernelci/kernelci-core/issues/2500

I'm considering adding something like `deb_arch`, `kernel_arch` and/or `kci_arch` to the platform configs (which would be automatically determined from `arch`, so we don't have to add more noise to the configs), that might solve a few things, but for now I'd suggest hardcoding the arch here.

Reply:
I missed that, thanks for the heads up! I hardcoded the arch for now.